modules/wri_search/config/schema/wri_search.schema.yml

```diff
@@ -0,0 +1,23 @@
+block.settings.single_file_component_block:wri_search_menu:
```

Comment:
This can actually work if we  don't use the component_context thing and just account for that in existing blocks.

Reply:
In truth, the problem was that "string" isn't translatable, it seems. But "text" is.